rtt/typekit/BoolTypeInfo.hpp

```diff
@@ -49,10 +49,10 @@ namespace RTT
         /**
          * Write boolean as 'true' or 'false'.
          */
-        struct BoolTypeInfo: public TemplateTypeInfo<bool,true>
+        struct BoolTypeInfo: public StdTypeInfo<bool>
         {
             BoolTypeInfo() :
-                TemplateTypeInfo<bool,true> ("bool")
+                StdTypeInfo<bool> ("bool")
             {
```

Comment:
I know that it's not ABI compatible, I just wonder if user code would be affected or not as this header is typically not include in user code, but is part of the rtt-typekit.